instagram/apps.py

```
@@ -4,3 +4,6 @@
 class InstagramConfig(AppConfig):
     default_auto_field = "django.db.models.BigAutoField"
     name = "instagram"
+
+    def ready(self):
+        import instagram.signals  # noqa: F401, PLC0415
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `F401`, `PLC0415`)

Remove unused `noqa` directive

(RUF100)